app/services/generic/searcher.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+class Generic::Searcher < Object
```

Comment:
I'm a bit confused by this class - I don't think we use `name` or `errors` anywhere, nor ActiveModel::Translation or ActiveModel::Validations, right? Is this going to be used for something later?

Reply:
Huh, did I wind up refactoring to not use errors, I guess I did. The Translation and Validations are iirc just so errors functions properly.

Reply:
Ah ok! I don't think `name` is used here either, right?

Makes sense that we'll want this later if/when we ever see errors in searches that we want to pass up to the controller, but I guess none of these fields are necessary right now

Reply:
`:name` is also for errors iirc, I think at some point i worked out the minimal viable setup for using activemodel errors.

Reply:
I basically just pulled this class from my old service PRs tho xD